geometry_msgs/Pose pose
float64 cartesian_stiffness
float64 rotational_stiffness